pytorch_lightning/utilities/parsing.py

```diff
@@ -196,9 +196,11 @@ def __repr__(self):
         return out
 
 
-def lightning_get_all_attr_holders(model, attribute):
-    """ Special attribute finding for lightning.  Gets all of the objects or dicts that holds attribute.
-            Checks for attribute in model namespace, the old hparams namespace/dict, and the datamodule. """
+def _lightning_get_all_attr_holders(model, attribute):
+    """
+    Special attribute finding for lightning. Gets all of the objects or dicts that holds attribute.
+    Checks for attribute in model namespace, the old hparams namespace/dict, and the datamodule.
```

Comment:
```suggestion
    Special attribute finding for Lightning. Gets all of the objects or dicts that holds attribute.
    Checks for attribute in model namespace, the old hparams namespace/dict, and the datamodule.
```
for consistency with the rest of the docs

Reply:
Updated!